youtube_dl/extractor/bandcamp.py

```diff
@@ -415,3 +408,52 @@ def _real_extract(self, url):
             'episode_id': compat_str(video_id),
             'formats': formats
         }
+
+
+class BandcampUserIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:user'
+    _VALID_URL = r'https?://(?:(?P<subdomain>[^.]+)\.)?bandcamp\.com/?$'
+
+    _TESTS = [{
+        'url': 'https://adrianvonziegler.bandcamp.com',
+        'info_dict': {
+            'id': 'adrianvonziegler',
+            'title': 'Albums of adrianvonziegler',
+        },
+        'playlist_mincount': 20,
+    }, {
+        'url': 'http://dotscale.bandcamp.com',
+        'info_dict': {
+            'id': 'dotscale',
+            'title': 'Albums of dotscale',
+        },
+        'playlist_count': 1,
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        uploader = mobj.group('subdomain')
+
+        webpage = self._download_webpage(url, uploader)
+
+        album_elements = re.findall(r'<a href="/album/(.+)">', webpage)
+
+        if not album_elements:
+            raise ExtractorError('The page doesn\'t contain any albums')
+
+        entries = [
+            self.url_result(
+                compat_urlparse.urljoin(url, 'album/{}'.format(album_id)),
```

Comment:
`{}` won't work in python 2.6.

Reply:
Fixed in 7eb84fd10528f3229052f896fe50fca3db101451

Reply:
More changes on this comment on ed4dd6a